### PR TITLE
Fixup tests for new MessageStreamer

### PR DESCRIPTION
**Pull request by mbonneau**

### CODE REVIEW

File: tests/TestCase.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Ratchet\RFC6455\Test;
+
+class TestCase extends \PHPUnit_Framework_TestCase
```
Comment (cboden) [inline tests/TestCase.php]: What's the purpose of this class?

Comment (mbonneau) [inline tests/TestCase.php]: There is no purpose right now. When I have setup testing on other projects, I sometimes end up using it to put common test stuff in there. I can remove it - just let me know.

Comment (cboden) [inline tests/TestCase.php]: Yes please. If that functionality is needed later we can add it back in then.

**Comment by mbonneau** [comment]
TestCase has been removed.

**Comment by cboden** [comment]
Merged, thanks! I enabled Travis after I merged, future commits/pulls should trigger the tests.

**Comment by cboden** [comment]
Looks so good! [Build Status](https://travis-ci.org/ratchetphp/RFC6455)

**Pull request merged**